official/utils/arg_parsers/parsers.py

```diff
@@ -201,6 +226,31 @@ def __init__(self, add_help=False, num_parallel_calls=True, inter_op=True,
           metavar="<MTS>"
       )
 
+    if dtype:
+      self.add_argument(
+          "--dtype", "-dt",
+          default="fp32",
+          choices=["fp16", "float16", "fp32", "float32"],
```

Comment:
This is nice, but also 2x the maintenance. Should we just enforce one nomenclature or the other?

Reply:
I think it's fine simply because the first thing we do is convert them out of strings. But then again I'd write a 1000 page opus in defense of shaving off a single character from a CLI arg. I'm going to leave it for now, but if you decide to decree I won't fight it.